pkg/providers/cloudstack/config/secret.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: eksa-system
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: eksa-license
+  namespace: eksa-system
+stringData:
+  license: "{{.eksaLicense}}"
```

Comment:
is this file used anywhere? can we remove it?

Reply:
We dont use this anywhere functional yet, but this is used when the support bundle is run.

Reply:
It's stored as vsphere only because this is not a file we need for docker as that is for experimental purposes, but if we want to go with a license approach for cloudstack, we would apply it here like we do for vsphere. Probably need to generalize it more, but I am assuming this can be a follow up action item to address based on if we even have a license to give people for this provider

https://github.com/aws/eks-anywhere/blob/main/pkg/providers/vsphere/vsphere.go#L1028